scanners/reactnative/reactnative.go

```diff
@@ -15,26 +15,31 @@ import (
 const scannerName = "react-native"
 
 const (
-	// workDirInputKey is a key of the working directory step input.
-	workDirInputKey = "workdir"
-)
+	projectDirInputTitle   = "React-native/Expo project directory"
```

Comment:
Do we need the Expo word here? Expo has its own scanner + Expo is built around RN projects.

Reply:
Not really, updated.